packages/core/core/src/AssetGraph.js

```diff
@@ -326,4 +381,37 @@ export default class AssetGraph extends Graph<AssetGraphNode> {
 
     return referenceId;
   }
+
+  resolveSymbol(asset: Asset, symbol: Symbol): SymbolResolution {
```

Comment:
this also is very specific to symbols and I'm not sure it should live directly on the `AssetGraph`. Maybe make this a function that takes an AssetGraph? Or maybe directly on the public Bundle object?

Reply:
One option I explored was to have symbol nodes in the asset graph, but it got complicated very quickly. Maybe a separate symbol graph? Either way, I'm not happy with the current approach of storing symbols in maps on assets/dependencies so any suggestions to make it more general/better would be helpful.